src/Api/createRequest.ts

```diff
@@ -111,9 +111,9 @@ export default async function createRequest<P = any>(
         // Since we also throw when response is not ok, re-throw the response data if available.
         const {
             status,
-            statusText = error.message || NETWORK_PROBLEM_ERROR_MESSAGE,
+            statusText = (error as any).message || NETWORK_PROBLEM_ERROR_MESSAGE,
```

Comment:
Looks like this package is built with old TS version where `error` in catch has `any` type